now.json

```diff
@@ -1,6 +1,5 @@
 {
   "version": 2,
```

Comment:
if the build script blows it away, does it even make sense to have this checked in? seems like it'd add confusion if you were trying to determine where the secrets came from via code inspection (i.e. ... me, yesterday)